src/components/Multiselect/Multiselect.vue

```diff
@@ -353,6 +356,7 @@ export default {
 	data() {
 		return {
 			elWidth: 0,
+			ariaExpanded: 'false',
```

Comment:
Should be a boolean. Above as well.

Reply:
in this case no, i could create ariaExpanded as Boolean, but then have to convert it to String within a template (ariaExpanded.toString())
please have a look https://v3-migration.vuejs.org/breaking-changes/attribute-coercion.html#overview

Reply:
Exactly, see my other comment https://github.com/nextcloud/nextcloud-vue/pull/3037#pullrequestreview-1072402046
I think that's more clear. Otherwise people might wonder why `false` is a string here.
Or is there anything wrong with having `toString()` in the template. I couldn't see anything like this in the link you posted.